rotkehlchen/tests/unit/test_defi_oracles.py

```diff
@@ -23,7 +23,7 @@
 
 @pytest.mark.parametrize('use_clean_caching_directory', [True])
 @pytest.mark.parametrize('should_mock_current_price_queries', [False])
-def test_uniswap_oracles_asset_to_asset(inquirer_defi):
+def test_uniswap_oracles_asset_to_asset(inquirer_defi, socket_enabled):  # pylint: disable=unused-argument  # noqa: E501
```

Comment:
Hey @yabirgb I was not sure about these 3 tests. I marked them as `socket_enabled` which means that they should not fail even when running with socket-disabled, just because I was not sure if they can be mocked.

They probably should, maybe except the first one? I need help here to determine what to do.